exporter/awsemfexporter/metric_translator.go

```diff
@@ -20,17 +20,15 @@ import (
 	"reflect"
 	"time"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/cwlogs"
```

Comment:
This is not following the import grouping rules used by this repo and will fail the lint checks.  Revert the import changes to this file.

Reply:
Done too !! Thanks for your insight !!!